documentation/ldap-guide.html

```diff
@@ -1,287 +1,357 @@
-            your base DN would be <tt>cn=Users;dc=activedirectory,dc=jivesoftware,dc=com</tt>. If
-            you've customized where users are stored, you'll just need to replicate that folder
-            structure using LDAP syntax.
+    </nav>
+
+    <section id="intro">
+
+        <h2>Introduction</h2>
+
+        <p>
+            This document details how to configure your Openfire installation to use
+            an external directory such as Open LDAP or Active Directory. Integration with a directory
+            lets users authenticate using their directory username and password. Optionally, you can
+            configure Openfire to load user profile and group information from the directory. Any group in
+            Openfire can be designated as a shared group, which means that you can pre-populate user's
+            rosters using directory groups.
         </p>
-    </li>
-    <li><b>Administrator DN</b><br/><br/>
-        <p>By default, Active Directory does not allow anonymous LDAP connections. Therefore,
-            you'll need to enter the DN of a user that's allowed to connect to the server and read
-            all user and group data. Unless you've created a special user account for this
-            purpose, an easy choice is to use the built-in administrator account. By default,
-            the administrator DN is in the form <tt>cn=Administrator,dc=&lt;Your Domain&gt;</tt>.
-            Using our previous example,
-            <tt>cn=Administrator,cn=users,dc=activedirectory,dc=jivesoftware,dc=com</tt>.
+
+        <p>Topics that are covered in this document:</p>
+
+        <nav>
+            <ul>
+                <li><a href="#background">Background</a>
+                <li><a href="#configuration">Configuration</a>
+                <li><a href="#activedirectory">Working with Active Directory</a>
+                <li><a href="#custom-search-filter">Custom Search Filter</a>
+                <li><a href="#ctxFactory">Custom Initial Context Factory</a>
+                <li><a href="#connectionPool">Connection Pooling</a>
+                <li><a href="#vcard">LDAP vCard Integration</a>
+                <li><a href="#faq">LDAP FAQ</a>
+            </ul>
+        </nav>
+    </section>
+
+    <section id="background">
+
+        <h2>Background</h2>
+
+        <p>
+            LDAP (Lightweight Directory Access Protocol) has emerged as a dominant standard
+            for user authentication and for storage of user profile data. It serves as a
+            powerful tool for large organizations (or those organizations integrating many
+            applications) to simplify user management issues. Many LDAP servers are available,
+            such as <a href="https://www.openldap.org/">Open LDAP</a>,
+            <a href="https://www.microsoft.com/windowsserver2003/technologies/directory/activedirectory/">Active Directory</a>,
+            and Novell's <a href="https://www.novell.com/products/edirectory/">eDirectory</a>.
```

Comment:
Remove this reference to Novell? Looks like it's been sold on, there's no trial that we can test with, and we've no idea if it'll work nowadays.

Reply:
Agreed.

Reply:
Fixed